testutils/memory.go

```diff
@@ -1,24 +1,44 @@
 package testutils
 
 import (
+	"fmt"
+
 	"github.com/src-d/metadata-retrieval/github/graphql"
 
 	"gopkg.in/src-d/go-log.v1"
 )
 
 // Memory implements the storer interface
 type Memory struct {
-	Organization *graphql.Organization
-	Repository   *graphql.RepositoryFields
-	Topics       []string
-	Users        []*graphql.UserExtended
-	PRs          []*graphql.PullRequest
-	PRComments   []*graphql.IssueComment
+	Organization     *graphql.Organization
+	Repository       *graphql.RepositoryFields
+	Topics           []string
+	Users            []*graphql.UserExtended
+	Issues           []*graphql.Issue
+	IssueComments    []*graphql.IssueComment
+	PRs              []*graphql.PullRequest
+	PRComments       []*graphql.IssueComment
+	PRReviews        map[int][]*graphql.PullRequestReview
+	PRReviewComments map[int]map[int][]*graphql.PullRequestReviewComment
+}
+
+// CountPRReviewsAndReviewComments returns the number of PR reviews and PR review comments
+func (s *Memory) CountPRReviewsAndReviewComments() (int, int) {
+	counter := 0
+	counter2 := 0
+	for k, v := range s.PRReviews {
+		counter += len(v)
+		for _, vv := range s.PRReviewComments[k] {
+			counter2 += len(vv)
+		}
+	}
+	return counter, counter2
 }
 
 // SaveOrganization stores an organization in memory,
 // it also initializes the list of users
 func (s *Memory) SaveOrganization(organization *graphql.Organization) error {
+	fmt.Printf("organization data fetched for %s\n", organization.Login)
 	log.Infof("organization data fetched for %s\n", organization.Login)
```

Comment:
IMO it's better to leave only `log.Infof` because tests do not need to spam logs unless it's debugging